apps/full-site-editing/full-site-editing-plugin/full-site-editing/store/selectors.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { createRegistrySelector } from '@wordpress/data';
+import { get } from 'lodash';
+
+export const isFullSitePage = createRegistrySelector( select => () => {
+	const { getBlocks, getCurrentPostType, getEditedPostAttribute } = select( 'core/editor' );
+
+	const postType = getCurrentPostType();
+	const templateId = get( getEditedPostAttribute( 'meta' ), '_wp_template_id' );
+	const blocks = getBlocks();
+
+	const isPage = postType === 'page';
```

Comment:
I did a bit of testing and it looks like in master, we're currently showing header/footer for post editing in addition to pages. I'll add a follow up issue to remind ourselves to fix that.